lib/utils/hmac-validator.ts

```diff
@@ -7,6 +7,8 @@ import {safeCompare} from '../auth/oauth/safe-compare';
 
 import ProcessedQuery from './processed-query';
 
+const HMAC_TIMESTAMP_PERMITTED_CLOCK_TOLERANCE_SEC = 10;
```

Comment:
Also, (sorry) are we sure 10s is enough for this, with slow networks and such? I think we could be a little more lenient, maybe a minute or two?

Reply:
I wasn't too sure, I used the same value as we used for Json validation. But in this case we may have a processing queue that is slow you are right. I'll put it to 90sec then Hopefully that should be enough. 
I am also conscious hat too much leniency leaves room for more vulnerability...

Reply:
Agreed on all counts, I think 90s is a reasonable compromise. 60 would probably be enough.